### ISSUE TITLE
Decorator unique_bits returns a None object

### ISSUE THREAD

**Issue by Paebbels** [pasztorpisti/py-flags#9]
When decorating a class derived from `Flags` with `unique_bits`, a None object is returned and a call to e.g. `HTTPMethods.GET` causes an attribute error, because `GET` is not a member of `None`.

**Example code:**

```python
from flags import Flags, unique_bits

@unique_bits
class HTTPMethods(Flags):
  """A list of supported HTTP methods."""
  GET =      1
  POST =     2
  PUT =      4
  PATCH =    8
  DELETE =  16
  HEAD =    32
  OPTIONS = 64
```

**Usage:**

```python
m = HTTPMethods.GET
```

**Error message:**

```python
AttributeError: 'NoneType' object has no attribute 'GET'
```

**Environment:**
* Python: 3.7.2
* py-flags: 1.1.2

**Comment by pasztorpisti** [comment]
The newly uploaded v1.1.3 fixes the issue.

Thanks for the report. It was an easy-to fix trivial bug. It could probably hide only because the decorator wasn't popular and no-one used it before. (And my tests didn't cover some basic things.)

(Note: I forgot to create a separate branch for the fix because I haven't worked on this repo for a while and the bug was so tiny that it could be fixed in no time.)

**Comment by Paebbels** [comment]
The issue is solved in v1.1.3.
